app/drivers/kscan/kscan_gpio_matrix.c

```diff
@@ -207,25 +220,22 @@ static int kscan_gpio_config_interrupts(const struct device **devices,
         return 0;                                                                                  \
     };                                                                                             \
     static int kscan_gpio_enable_##n(const struct device *dev) {                                   \
-        COND_CODE_1(CONFIG_ZMK_KSCAN_MATRIX_POLLING,                                               \
-                    (struct kscan_gpio_data_##n *data = dev->data;                                 \
-                     k_timer_start(&data->poll_timer, K_MSEC(10), K_MSEC(10)); return 0;),         \
-                    (int err = kscan_gpio_enable_interrupts_##n(dev);                              \
-                     if (err) { return err; } return kscan_gpio_read_##n(dev);))                   \
+        COND_POLL_OR_INTERRUPTS((struct kscan_gpio_data_##n *data = dev->data;                     \
+                                 k_timer_start(&data->poll_timer, K_MSEC(10), K_MSEC(10));         \
```

Comment:
Am I correctly interpreting this that when polling is enabled, we poll at a constant 100 Hz? It might be useful to make that configurable, for example if someone wanted to make a wired keyboard that polls as fast as possible.

Reply:
Yeah, I hadn't remembered hardcoding.this, but it does seem so, yeah. Good item to move into Kconfig soon, yeah.